bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -201,7 +203,14 @@ const getUsers = () => {
   return users.sort(sortUsers);
 };
 
-const hasBreakoutRoom = () => Breakouts.find({ parentMeetingId: Auth.meetingID }).count() > 0;
+const hasBreakoutRoom = () => Breakouts.find({ parentMeetingId: Auth.meetingID },
+  { fields: {} }).count() > 0;
+
+const isMe = userId => userId === Auth.userID;
+const isUserModerator = (userId) => {
```

Comment:
This function is problematic. It's being used all over the place when it shouldn't be. Almost all of the uses that I've seen have been in components (and sometimes containers) that already have access to a User record and then they make secondary calls to this helper function that grabs a record the components already have. It's one of the reasons that it's recommended to never pass service functions to the components because it's too easy to abuse. All these extra calls will lead to performance issues.